hudi-client/src/main/java/org/apache/hudi/table/upgrade/UpgradeDowngrade.java

```diff
@@ -27,6 +27,7 @@
 import org.apache.hadoop.fs.FSDataOutputStream;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.fs.Path;
+import org.apache.hudi.exception.HoodieUpgradeDowngradeException;
```

Comment:
It would be better to add an empty line to split the different type of import statements?

Reply:
> It would be better to add an empty line to split the different type of import statements?
@yanghua thanks for your review. I moved it to be with other hudi import statements.